### PR TITLE
Add IDs to some commpanel elements so <label>s actually work

### PR DESCRIPTION
**Pull request by betaveros**

### CODE REVIEW

**Comment by betaveros** [comment]
Wait whoaaa we have duplicate names/IDs? I had this commit lying around so I pushed it without thinking

**Comment by benjaminjkraft** [comment]
Yeah -- they're in separate forms, but we do :cry:.  So you'll probably have to change them to do this.

File: esp/templates/users/usersearch/list_selector.html [hunk 1]
```diff
@@ -149,16 +149,16 @@
             </select>
             <div class="sendto_fn_select Student">
                 <div>
-                    <input type="checkbox" name="student_sendto_self" value="1" checked="checked" />
-                    <label for="student_sendto_self">Email the students</label>
+                    <input type="checkbox" name="combo-student_sendto_self" id="combo-student_sendto_self" value="1" checked="checked" />
```
Comment (benjaminjkraft) [inline esp/templates/users/usersearch/list_selector.html]: Is the name change here intentional?  Won't that break the form?

**Comment by willgearty** [comment]
What's the status on this PR @betaveros ?

**Comment by willgearty** [comment]
This was also implemented as part of #3424.

**Pull request closed without merging**